site-builder/src/site/resource.rs

```diff
@@ -373,6 +387,7 @@ impl ResourceManager {
             content_type,
             *content_encoding,
             output.blob_id,
+            U256::from_le_bytes(&blob_hash),
```

Comment:
Question: any specific reason behind the use of `le` bytes?

Reply:
That was the only way I could find to cast the byte array to U256.

Reply:
ah yes, now I recall that BCS encodes as LE bytes.